src/adagrams.js

```diff
@@ -1,15 +1,107 @@
+// global variable
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
```

Comment:
Great job creating this global variable outside of each function so each function can access it & we can also keep the program DRY 😄